packages/client/src/app/views/InfoView.vue

```diff
@@ -1,231 +1,261 @@
+<script setup>
+const intro = {
+  title: 'Plastic Problemen!',
+  subtitle: 'Leer over plastic met onze Slime!'
+};
+
+const main = {
+  subtitle: 'Uitleg over plastic afval en het probleem!'
+};
+
+const footer = {
+  subtitle: 'Wist je dat?'
+};
+
+const register = {
+  subtitle: 'Wil je meer leren? Registreer je dan en daag jezelf uit!'
+};
```

Comment:
Is there a reason why you've put the texts for the `main`, `footer`, and `register` inside a Js object instead of directly putting them in the HTML?